core/services/job/orm.go

```diff
@@ -579,7 +579,7 @@ type DRSpecConfig interface {
 }
 
 func LoadEnvConfigVarsVRF(cfg DRSpecConfig, vrfs VRFSpec) *VRFSpec {
-	// Take the larger of the global vs specific.
+	// Take the largest of the global vs specific.
```

Comment:
"larger" is actually correct English here, since the comparative is preferred when comparing two objects (three or more would require "largest").

/grammar nerd

Reply:
I got this hint from my grammar/spelling check in Goland, hence the change. But thank you for letting me know the difference!